UefiPayloadPkg/BlSMMStoreDxe/BlSMMStoreFvbDxe.c

```diff
@@ -49,6 +49,7 @@ InitializeFvAndVariableStoreHeaders (
   IN SMMSTORE_INSTANCE *Instance
   )
 {
+  DEBUG((EFI_D_INFO, "%a\n", __FUNCTION__));

```

Comment:
Changes in this file are no longer needed?

Reply:
Deleted whole file